### DISCUSSION TITLE
How to sort folders?

### DISCUSSION THREAD

**Discussion by severinlandolt** [storybookjs/storybook#26635] [Help]
### Summary

Hi all! I have two folders:

![CleanShot 2024-03-24 at 20 32 48@2x](https://github.com/storybookjs/storybook/assets/41927988/54eeda7e-128b-470e-bb1d-3eca8d76aab9)

And I would like to sort them differently: 1. Visualization 2. UI

I found this: https://storybook.js.org/docs/writing-stories/naming-components-and-hierarchy#sorting-stories

But this is for stories only, can I custom sort folders in storybook?

**Comment by marksy** [comment]
Does this help?

```
options: {
        storySort: {
            order: ['Visualization', 'UI'],
        },
    },
```

**Reply by severinlandolt** [reply]
Unfortunately, no
